code/__DEFINES/hud.dm

```diff
@@ -36,3 +36,6 @@
 #define ANTAG_HUD_NINJA 14
 #define ANTAG_HUD_CHANGELING 15
 #define ANTAG_HUD_ABDUCTOR 16
+#define ANTAG_HUD_DEMON	17

```

Comment:
I was trying to keep all of the numbers aligned.  Look at the cult/rev/ops/wiz huds.